src/main/java/com/flipkart/ranger/finder/ServiceRegistryUpdater.java

```diff
@@ -139,14 +145,15 @@ public void checkForUpdate() {
     }
 
     public void stop() {
-        logger.debug("Stopped updater");
+        logger.info("Stopped updater for [{}]", serviceRegistry.getService().getServiceName());
```

Comment:
debug

Reply:
This is done once during the lifetime of registry. Corresponding start method also has similar message at INFO level